packages/internal-helpers/src/remote.ts

```diff
@@ -14,15 +21,37 @@ export function matchPattern(url: URL, remotePattern: RemotePattern) {
 	);
 }
 
-export function matchPort(url: URL, port?: string) {
+/**
+ * Checks if the given URL's port matches the specified port. If no port is provided, it returns true.
+ *
+ * @param {URL} url - The URL object whose port will be checked.
+ * @param {string} [port] - The port to match against the URL's port. Optional.
```

Comment:
Just wondering whether you already have a precedent for how you show optional parameters and suggesting that you be consistent with that that if it exists. 

In docs, for example, I think we tend to "front-load" the optional part, or even if we don't, it's usually lowercase and in parenthesis:

```
 * @param {string} [port] - (optional) The port to match against the URL's port.
```

Not suggesting you need to change this! Only a reminder that if you *do* have other optional params already written, it's nice if they're written the same way consistently!

Reply:
As per JSDoc, a parameter is optional when is surrounded in square brackets: https://jsdoc.app/tags-param#optional-parameters-and-default-values

In this example, the variable `port` is optional because it's written `[port]`. Unfortunately, the IDEs aren't very good at this, and they don't render it as optional

Reply:
OK, not opposed to adding `Optional.` at the end of the text if you think that's a helpful way to display it! Just wanted to see if you already had a convention to follow because it seems like there would be other optional values, too, and it's always great if things are consistent. I leave it totally up to you!